src/cider/nrepl/middleware/track_state.clj

```diff
@@ -22,13 +22,19 @@
 
 ;;; Auxiliary
 
+(defn meta-with-fn
```

Comment:
This is very specific to vars. I think it's best to rename the function to var-meta-with-fn.

Reply:
And make it private

Reply:
Ah, yes, you are right!